config/view.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
@webong @Mh-Asmi is this relevant/necessary for this project?

Reply:
Since we don't make use of Blade for mailing, it's sort of not needed. Actually the view config file doesn't really do anything.